plugins/modules/vm_snapshot_attach_disk.py

```diff
@@ -107,11 +107,15 @@
 from ..module_utils.client import Client
 from ..module_utils.rest_client import RestClient
 from ..module_utils.vm_snapshot import VMSnapshot
+from ..module_utils.vm import VM
 from ..module_utils.task_tag import TaskTag
 from ..module_utils.typed_classes import TypedDiff
 from typing import Tuple, Dict, Any, Optional
 
 
+# A minimum size of an IDE typed disk in HC3 API is 1 GB
+MIN_TYPE_IDE_SIZE = 1000341504  # bytes
```

Comment:
This is the minimum allowed size for disks of type IDE. If the size is not adjusted, then integration tests fail, because of an error (on HC3 API) when trying to reboot that VM. When I did some debugging, I found, that if I adjust the size to 1 GB (on HC3 API) by myself and then try to boot up that VM, the error is fixed...